src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -140,8 +140,8 @@ public Index parseIndexToEdit(String args) throws ParseException {
         Index index = null;
 
         try {
-            String indexStr = args.substring(2, 3);
-            index = ParserUtil.parseIndex(indexStr);
+            String[] splitArgs = args.split(" ", 3);
+            index = ParserUtil.parseIndex(splitArgs[1]);
         } catch (ParseException pe) {
             throw new ParseException(
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditPersonCommand.MESSAGE_USAGE), pe);
```

Comment:
Maybe we can replace the exception message with the one for invalid index?

Reply:
Unfortunately, we have to keep it as it was in v1.3 due to feature freeze issue.